.eslintrc.js

```diff
@@ -2300,6 +2300,23 @@ module.exports = {
         'no-console': 'off',
       },
     },
+    {
+      files: ['x-pack/**/*.ts'],
+      rules: {
+        'no-restricted-imports': [
+          'error',
+          {
+            paths: [
+              {
+                name: '@kbn/cypress-test-helper',
+                message:
+                  "Import from a sub-path (e.g. '@kbn/cypress-test-helper/src/utils'). Cypress uses Webpack, which requires direct file imports to avoid parse errors.",
+              },
+            ],
+          },
+        ],
+      },
+    },
```

Comment:
This rule prevents the issue I discovered with importing from test package:

>Cypress runs test and support files through a Webpack preprocessor. When importing from a package entry point, Webpack may try to parse files that aren’t valid JavaScript/TypeScript (like raw TypeScript, license headers, or markdown), leading to "Module parse failed" errors. Direct subpath imports (e.g. @kbn/cypress-test-helper/src/utils) ensure Webpack only processes actual code files it can handle, avoiding these issues